### PR TITLE
8296934: Write a test to verify whether Undecorated Frame can be iconified or not

### PR DESCRIPTION
**Pull request by janto001**
This test case verifies that Undecorated Frame can be iconified or not
Using mouse click events generated WINDOW_ACTIVATED, WINDOW_ICONIFIED, WINDOW_DEACTIVATED, and WINDOW_DEICONIFIED and tested whether it is iconified or not.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8296934](https://bugs.openjdk.org/browse/JDK-8296934): Write a test to verify whether Undecorated Frame can be iconified or not

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11812/head:pull/11812` \
`$ git checkout pull/11812`

Update a local copy of the PR: \
`$ git checkout pull/11812` \
`$ git pull https://git.openjdk.org/jdk pull/11812/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11812`

View PR using the GUI difftool: \
`$ git pr show -t 11812`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11812.diff">https://git.openjdk.org/jdk/pull/11812.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
>Integration blocker
> ⚠️ Failed to retrieve information on issue 8296934. Please make sure it exists and is accessible.

File: test/jdk/java/awt/Frame/Iconify/IconifyTest.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        });
+
+        frame.setBackground(Color.green);
+        frame.add(button);
+        frame.setLocationRelativeTo(null);
+        frame.toFront();
+        frame.setVisible(true);
+    }
+
+    public static void main(String[] args) throws AWTException, InvocationTargetException, InterruptedException {
```
Comment (mrserb) [inline test/jdk/java/awt/Frame/Iconify/IconifyTest.java:105]: Please split the long lines to have 80 chars per line.

Comment (janto001) [reply test/jdk/java/awt/Frame/Iconify/IconifyTest.java]: Done the changes

**Review by mrserb — APPROVED** [review]
Looks fine, I assume the code was tested in the mach5.

**Comment by janto001** [comment]
> Looks fine, I assume the code was tested in the mach5.

Yes, It's working fine in mach5

**Pull request closed without merging**